server/dash/dashboards/nc_summary_comparison.py

```diff
@@ -0,0 +1,842 @@
+    """
+    graph = html.Div(dcc.Graph(id="nc_avg_comp_line_chart", style={"height": "40vh",
+         "width": "97.4vw"}),
+        style=merge_dict(BORDER_STYLE, {"height": "40vh", "width": "97.4vw"}))
+    return graph
+
+
+def generate_summary_pie_chart():
+    """Generates the pie chart for the summary dashboard.
+
+    This functions generates the html elements for the 
+    request type pie chart for summary dashboard.
+
+    Return:
+        Dash html div element containing request type pie chart.
+    """
+    pie_chart = html.Div(
+        # Pie Chart for the share of request type.
+        dcc.Graph(
+            id="req_type_pie_chart", style={"height": "40vh",
+                            "width": "48.5vw"}
+        ), style=merge_dict(CHART_OUTLINE_STYLE, {"width": "48.5vw",
+                        "height": "40vh"})),  # for border-radius , add stuff later
+    return pie_chart
+
+
+def generate_summary_histogram():
+    """Generates the histogram for the summary dashboard.
+
+    This functions generates the html elements for the 
+    request time to close histogram for summary dashboard.
+
+    Return:
+        Dash html div element containing request time to close histogram.
+    """
+    histogram = html.Div(
+        # Histogram for the request timeToClose.
+        dcc.Graph(
+            id="time_close_histogram", style={"height": "40vh",
+                     "width": "48vw"}
+        ), style=merge_dict(CHART_OUTLINE_STYLE, {"width": "48vw",
+                 "height": "40vh"}))
+    return histogram
+
+
+def generate_council_name_dropdown():
+    """Generates the neighborhood council (nc) dropdown for the 
+    comparison dashboard.
+
+    This functions generates the html elements for the 
+    nc dropdown for the left side of the comparison dashboard.
+
+    Return:
+        Dash html div element containing nc drop down for left pane filtering.
+    """
+    dropdown = html.Div(dcc.Dropdown(sorted([n for n in set(api_data_df["councilName"])]),
+         " ", id="nc_comp_dropdown",
+                 placeholder="Select a Neighborhood Council..."),
+                 style=merge_dict(INLINE_STYLE, {"width": "48.5vw"})),
+    return dropdown
+
+
+def generate_council_name_dropdown2():
+    """Generates the neighborhood council (nc) dropdown for the 
+    comparison dashboard.
+
+    This functions generates the html elements for the 
+    nc dropdown for the right side of the comparison dashboard.
+
+    Return:
+        Dash html div element containing nc drop down for right pane filtering.
+    """
+    dropdown = html.Div(dcc.Dropdown(sorted([n for n in set(api_data_df["councilName"])]),
+         " ", id="nc_comp_dropdown2",
+                 placeholder="Select a Neighborhood Council..."),
+                 style=merge_dict(INLINE_STYLE, {"width": "48.5vw"}))
+    return dropdown
+
+
+def generate_comparison_total_req():
+    """Generates the indicator visual for the 
```

Comment:
Could you clarify what indicator visual means here?

Reply:
Like just a number/figure showing on the card.